src/jailer/src/main.rs

```diff
@@ -136,6 +141,9 @@ impl fmt::Display for Error {
             GetOldFdFlags(ref err) => write!(f, "Failed to get flags from fd: {}", err),
             Gid(ref gid) => write!(f, "Invalid gid: {}", gid),
             InvalidInstanceId(ref err) => write!(f, "Invalid instance ID: {}", err),
+            Metadata(ref path, ref err) => {
```

Comment:
I must be doing smth wrong, I do not see this used anywhere...

Reply:
Oops, leftover. Removed.